src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/InitialTestProcessTests.cs

```diff
@@ -27,10 +27,10 @@ public InitialTestProcessTests()
         public void InitialTestProcess_ShouldThrowExceptionOnFail()
         {
             var testRunnerMock = new Mock<ITestRunner>(MockBehavior.Strict);
-            var test1 = Guid.NewGuid();
-            var test2 = Guid.NewGuid();
-            var ranTests = new TestsGuidList(new[] {test1, test2});
-            var failedTests = new TestsGuidList(new[] {test1});
+            var failedTest = Guid.NewGuid();
+            var successfulTest = Guid.NewGuid();
+            var ranTests = new TestsGuidList(new[] {failedTest, successfulTest});
```

Comment:
I'm now thinking, maybe we could add a constructor for TestsGuidList that takes a params param? That way we don't need to create a array for each list.

Reply:
done, but it turns out it is only used 4 times. Maybe it will be helpful in the future